Build/TileEngine/Radar_Screen.cc

```diff
@@ -278,10 +269,10 @@ void RenderRadarScreen()
 			INT16 const sHeight = RADAR_WINDOW_HEIGHT;
 			INT16 const sX      = RADAR_WINDOW_X;
 
-			INT16 const sRadarTLX = sTopLeftWorldX     * gdScaleX - sRadarCX + sX       + sWidth  / 2;
-			INT16 const sRadarTLY = sTopLeftWorldY     * gdScaleY - sRadarCY + gsRadarY + sHeight / 2;
-			INT16 const sRadarBRX = sBottomRightWorldX * gdScaleX - sRadarCX + sX       + sWidth  / 2;
-			INT16 const sRadarBRY = sBottomRightWorldY * gdScaleY - sRadarCY + gsRadarY + sHeight / 2;
+			INT16 const sRadarTLX = sTopLeftWorldX     * gdScaleX - sRadarCX + sX      					 + sWidth  / 2;
```

Comment:
yes. i Had to adjust the tabbing to reflect hte new length of the lines. Looks fine when viewed with 2 spaces as tabs.